### PR TITLE
Third task

### PR DESCRIPTION
**Pull request by EduardoAquinta**
Added GET category functionality and appropriate error handling.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data")
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+
+//closes all databases after the test suite has run.
+afterAll(() => {
+    if (db.end) db.end();
```
Comment (Augs0) [inline __tests__/app.test.js:9]: We don't need the if statement here. It's fine just to return the `db.end()`

File: controllers/boardgames-controllers.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const express = require("express");
```
Comment (Augs0) [inline controllers/boardgames-controllers.js:1]: I'm a bit confused why these three imports are here. The only thing we need to import into the controller is our model functiona

**Review by Augs0 — APPROVED** [review]
Great job, Eddie. A few things you can cut out, but otherwise the code looks ok to me  👍 feel free to merge once you've read through and acted on the comments

File: __tests__/app.test.js (hunk 1 above)
Comment (EduardoAquinta) [reply __tests__/app.test.js:9]: ok cool I'll simplify it.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+const app = express();
+
+const { 
+    getMessage,
+    getCategory
+} = require("./controllers/boardgames-controllers");
+
+//app.use(express.json()); ???
```
[the remark this answers is not in the document]
Comment (EduardoAquinta) [reply app.js:9]: Will leave it alone!

File: controllers/boardgames-controllers.js (hunk 2 above)
Comment (EduardoAquinta) [reply controllers/boardgames-controllers.js:1]: I'm a bit confused too! So I don't need to import 'express'?

**Comment by EduardoAquinta** [comment]
Thanks August, that's made me feel like I'm getting somewhere!

**Pull request merged**